doc/changelog.md

```diff
@@ -46,114 +139,169 @@
-  and execute multiple programs in a multithreaded environment.
-  [(#381)](https://github.com/PennyLaneAI/catalyst/pull/381)
+  ```pycon
+  >>> dev = qml.device("lightning.qubit", wires=2, shots=100)
+  >>> @qjit
+  >>> @qml.qnode(dev)
+  >>> def circuit(x):
+  >>>     qml.RX(x, wires=0)
+  >>>     return qml.probs(wires=0)
+  >>> circuit(0.54)
+  array([0.94, 0.06])
+  >>> circuit(0.54)
+  array([0.93, 0.07])
+  ```
 
-* Qjitted functions now support asynchronuous execution of QNodes. Simply use ``qjit(async_qnodes=True)`` to
-  enable the async execution of QNodes. It is useful for finite differences and parameter-shift, as those
-  differentiation methods can generate multiple circuit. Support for the async MLIR dialect was added.
-  [(#374)](https://github.com/PennyLaneAI/catalyst/pull/374)
+* Catalyst gradient functions `grad`, `jacobian`, `jvp`, and `vjp` can now be invoked from
+  outside a `@qjit` context.
+  [(#375)](https://github.com/PennyLaneAI/catalyst/pull/375)
+
+  This simplifies the process of writing functions where compilation
+  can be turned on and off easily by adding or removing the decorator. The functions dispatch to
+  their JAX equivalents when the compilation is turned off.
 
-  In this example below, the first and second circuit are executed in parrallel if two threads are available.
-  To see a speed up in your code, you should use circuits with more gates and/or more qubits.
   ```python
   dev = qml.device("lightning.qubit", wires=2)
 
-  @qjit(async_qnodes=True)
-  def multiple_qnodes(params):
-      @qml.qnode(device=dev)
-      def circuit1(params):
-          qml.RX(params[0], wires=0)
-          qml.RY(params[1], wires=1)
-          qml.CNOT(wires=[0, 1])
-          return qml.expval(qml.PauliZ(wires=0))
-
-      @qml.qnode(device=dev)
-      def circuit2(params):
-          qml.RY(params[0], wires=0)
-          qml.RZ(params[1], wires=1)
-          qml.CNOT(wires=[0, 1])
-          return qml.expval(qml.PauliX(wires=0))
-
-      @qml.qnode(device=dev)
-      def circuit3(params):
-          qml.RZ(params[0], wires=0)
-          qml.RX(params[1], wires=1)
-          qml.CNOT(wires=[0, 1])
-          return qml.expval(qml.PauliZ(wires=0))
-
-      new_params = jnp.array([circuit1(params), circuit2(params)])
-      return circuit3(new_params)
-  ```
-  ``` pycon
-  >>> func(jnp.array([1.0, 2.0]))
-  1.0
+  @qml.qnode(dev)
+  def circuit(x):
+      qml.RX(x, wires=0)
+      return qml.expval(qml.PauliZ(0))
   ```
 
-<h3>Improvements</h3>
+  ```pycon
+  >>> grad(circuit)(0.54)  # dispatches to jax.grad
+  Array(-0.51413599, dtype=float64, weak_type=True)
+  >>> qjit(grad(circuit))(0.54). # differentiates using Catalyst
+  array(-0.51413599)
+  ```
 
-* Support for `mcmc` sampling in `lightning.qubit`.
+* All `lightning.qubit` configuration options are now supported via the `qml.device` loader,
+  including Markov Chain Monte Carlo sampling support.
```

Comment:
@erick-xanadu is this correct? E.g., is the support for parallel observable measurements via the lightning `batch_obs=True` working?

Reply:
I am not sure `batch_obs=True`. What this line says to me is that `mcmc=True` and `num_burnin` and `kernel_name` options are available and the measurements happen via the `generate_samples_metropolis` function. Maybe @maliasadi can help me understand better if what we are both saying is the same thing?

Reply:
@josh146  @erick-xanadu 
- `mcmc` : it's Markov Chain Monte Carlo sampling support (it's implemented in the C++ API of Lightning)
- `batch_obs` : it calculates jacobians for chucks of observables (up to my understanding this is mainly in Python and added to the Python API of Lightning)

Anyways, we don't support `batch_obs` at the moment.

Reply:
```suggestion
* New `lightning.qubit` configuration options are now supported via the `qml.device` loader,
  including Markov Chain Monte Carlo sampling support.
```